iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/consensus/index/impl/MinimumProgressIndex.java

```diff
@@ -79,6 +81,11 @@ public ProgressIndexType getType() {
     return ProgressIndexType.MINIMUM_PROGRESS_INDEX;
   }
 
+  @Override
+  public TotalOrderSumTuple getTotalOrderSumTuple() {
+    return new TotalOrderSumTuple(0L);
```

Comment:
```suggestion
    return new TotalOrderSumTuple(0L);
```

private static final TotalOrderSumTuple TOTAL..TUPLE = new TotalOrderSumTuple(Long.MIN_VALUE);

Reply:
negative value of MinimumProgressIndex is forbidden for the correctness of algorithm.